### PR TITLE
Renames in module and decoder registries inspired by @mxz42

### PR DESCRIPTION
**Pull request by elsirion**
Renames in module and decoder registries inspired by @mxz42

### CODE REVIEW

**Comment by justinmoon** [comment]
these all look like improvements to me!

**Review by dpc — APPROVED** [review]

File: fedimint-api/src/module/registry.rs [hunk 1]
```diff
@@ -58,16 +58,17 @@ impl ServerModuleRegistry {
...
-    pub fn new(decoders: impl IntoIterator<Item = (ModuleKey, Decoder)>) -> ModuleDecoderRegistry {
-        ModuleDecoderRegistry(decoders.into_iter().collect())
-    }
-
     /// Return the decoder belonging to the module identified by the supplied `module_key`
     ///
     /// # Panics
     /// If the decoder isn't in the registry
-    pub fn decoder(&self, module_key: ModuleKey) -> &Decoder {
+    pub fn get_decoder(&self, module_key: ModuleKey) -> &Decoder {
```
Comment (Maan2003) [inline fedimint-api/src/module/registry.rs:65]: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

:shrug:

Comment (elsirion) [reply fedimint-api/src/module/registry.rs:65]: So … just `get(…)` I guess?

File: fedimint-api/src/module/registry.rs [hunk 2]
```diff
@@ -40,7 +40,7 @@ impl<M> ModuleRegistry<M> {
     ///
     /// # Panics
     /// If the module isn't in the registry
-    pub fn module(&self, module_key: ModuleKey) -> &M {
+    pub fn get_module(&self, module_key: ModuleKey) -> &M {
```
Comment (Maan2003) [inline fedimint-api/src/module/registry.rs:43]: missed this one

**Review by Maan2003 — APPROVED** [review]

**Comment by elsirion** [comment]
I think after the renames it becomes worthwhile to combine the decoder and module registries.

**Review by dpc — APPROVED** [review]

**Pull request merged**